org/readme.md

```diff
@@ -63,6 +63,12 @@ branch-protection:
           include: [ "^<default branch>$", "^v[0-9]*$"]  # note the surrounding ^...$ to avoid matching branches containing 'main' or 'v'
 ```
 
+Best Practices:
+- Replace github deploy keys by working group bot users. Branch protection rules enforce PRs for commits with deploy keys (enforce_admins=true).
+- Ensure that all bot users are members of the working group (area) bots team.
```

Comment:
This is something automation takes care of right? As in if they get added to the `bots:` section they will be added to all areas? The `(area)` bit in this statement is the confusing part.

Reply:
Yes. WG bots team is added to all repos of all WG areas.

ARD had a bot user that was added as direct user to some repos. This did not work because only the bots teams are excluded.

Bots users can be added to the complete WG or to an area. They get added by the automation to the teams wg-<wg name>-bots or wg-<wg name>-<area name>-bots. Both bots teams are excluded from branch protection. WG area bots team is optional.

Maybe better: Ensure that all bot users are members of the working group bots or working group area bots team.